config/urls.py

```diff
@@ -1,19 +1,5 @@
-"""config URL Configuration
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/3.0/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  path('', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.urls import include, path
-    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
-"""
-from django.urls import path
+from django.urls import path,include
```

Comment:
```suggestion
from django.urls import (
    path,
    include
)
```
같은 줄에서 import 하고자 하는 것이 두가지 이상일 때, 개별 줄로 분리해주는 것이 위코드 코딩 컨벤션입니다 :)